samples/JwtBearerSample/Startup.cs

```diff
@@ -59,7 +57,7 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     OnAuthenticationFailed = c =>
                     {
-                        c.HandleResponse();
+                        c.Ignore();
```

Comment:
> I'm not even sure why we need to have this.

We're really repeating the same things again and again :sweat_smile: 
This sample demonstrates one important scenario: skipping the default logic and taking control of the challenge logic (e.g to return a JSON payload, like what FB does for failed requests).